tools/cli/commands.js

```diff
@@ -1579,8 +1579,31 @@ function doTestCommand(options) {
   // cleaned up on process exit. Using a temporary app dir means that we can
   // run multiple "test-packages" commands in parallel without them stomping
   // on each other.
-  var testRunnerAppDir =
-        options['test-app-path'] || files.mkdtemp('meteor-test-run');
+  let testRunnerAppDir;
+  const testAppPath = options['test-app-path'];
+  if (testAppPath) {
+    try {
+      if (files.mkdir_p(testAppPath, 0o700)) {
```

Comment:
Is there really a need to specify umask? Wouldn't environment's umask apply if somebody wants to configure this? I have not seen elsewhere in Meteor that we would specify this.